adobe/react-spectrum
# Add position: relative to fix the checkbox position.

Closes 

## ✅ Pull Request Checklist:


## 📝 Test Instructions:
The [ComboBox's sample CSS](https://react-spectrum.adobe.com/react-aria/ComboBox.html#example) is causing wrong checkbox positioning when implemented.

To replicate:
- Copy the sample CSS to a project and use the ComboBox component.
- Select other values, and you will notice that the checkbox is still pointing to the first item.

 For example:
![image](https://github.com/user-attachments/assets/df78ca26-9c75-4a58-a8ab-f12aaf755f02)
